### DISCUSSION TITLE
Circular pool fire tank using pyrosim

### DISCUSSION THREAD

**Discussion by PurnaPushkalaSomasundaram** [firemodels/fds#12770] [Q&A]
Hello! My name is Purna Pushkala Somasundaram from Politecnico di Milano. My final thesis is about corn oil pool fires of 0.8 m diameter using FDS. I am using Pyrosim to model it.  have modelled with square fire unit obstruction and got very good results. Is it possible to make a circular tank using pyrosim? If so, can anyone help me through this? How  to  change the rectangular  fire obstruction to a circular one using pyrosim?

[2af_git.txt](https://github.com/firemodels/fds/files/14928966/2af_git.txt)

**Comment by PurnaPushkalaSomasundaram** [comment]
I will upload the file again as that file is not correct.
[2af.txt](https://github.com/firemodels/fds/files/14929031/2af.txt)

**Comment by rmcdermo** [comment]
It is possible.  But I suggest you contact Thunderhead user support.

https://support.thunderheadeng.com/pyrosim

**Reply by PurnaPushkalaSomasundaram** [reply]
Thank you
